src/vue/backend/src/templates/importHeader.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <idx-tab-container
+        customClass="import-header"
+        @activeTab="switchTabs"
+        :activeTab="activeTab"
+        :tabs="['Unimported', 'Imported']"
+    >
+        <idx-block className="import-header__description">{{ description }}</idx-block>
+        <idx-block className="import-header__actions-bar">
+            <idx-block className="import-header__select-all" @click="$emit('select-all', selected)">{{ selected ? 'Select All' : 'Deselect All' }}</idx-block>
+            <idx-button customClass="import-header__action" @click="$emit('bulk-action', action)">{{ action }} Selected</idx-button>
+        </idx-block>
+        <router-view></router-view>
+    </idx-tab-container>
+</template>
+<script>
+export default {
+    name: 'import-header',
+    data () {
+        return {
+            activeTab: 0
+        }
+    },
+    props: {
+        selected: {
+            type: Boolean,
+            default: false
+        },
+        action: {
+            type: String,
+            default: 'Import'
+        },
+        description: {
+            type: String,
+            default: ''
+        }
+    },
+    methods: {
+        switchTabs (e) {
+            this.activeTab = e
+            /* Router change here, once we have router set up we can switch it. For now emitting and event */
+            this.$emit('switch-tabs', this.activeTab)
+        }
+    }
+}
+</script>
+<style lang="scss">
+@import '~@idxbrokerllc/idxstrap/dist/styles/components/buttons';
+@import '~@idxbrokerllc/idxstrap/dist/styles/components/tabContainer';
+.import-header {
+    background-color: $white;
```

Comment:
I think if you add `font-size: 1rem;` to the `.import-header` selector, all child elements would inherit so you could write fewer font size declarations.